### ISSUE TITLE
[Preemptive] Box shadows on forms might get nasty

### ISSUE THREAD

**Issue by csswizardry** [sky-uk/toolkit-ui#27]
`box-shadow` is super expensive: it causes layout, render, and composite in Blink, Webkit, and Gecko. It causes paint and composite in Edge.

There aren’t any problems I can prove just yet (they are measurable, but they aren’t problems), but if we get longer pages of forms in which all of our fields are toggling shadows on and off, we’re asking a lot of the browser. Recalculating all of that stuff every time we enter or leave an input is going to get noticable.

Basically, what I’m preemptively saying is: **we should try not to get attached to `box-shadow`s.** They could get pretty costly and cause huge frustration for users.

Let’s keep an eye on it and be prepared to remove them if we need to.

**Comment by joebell93** [comment]
Going to close this for now until it becomes more of an issue
